rest/client/rest.go

```diff
@@ -20,46 +20,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (*communicatorImpl) GetAllHosts() {}
-func (*communicatorImpl) GetHostByID() {}
-
-// GetHostsByUser will return a slice of all hosts spawned by the given user
-// The API route is paginated, but we will add all pages to a local slice because
-// there is an application-defined limit on the number of hosts a user can have
-func (c *communicatorImpl) GetHostsByUser(ctx context.Context, user string) ([]*model.APIHost, error) {
-	info := requestInfo{
-		method:  get,
-		path:    fmt.Sprintf("/users/%s/hosts", user),
-		version: apiVersion2,
-	}
-
-	p, err := newPaginatorHelper(&info, c)
-	if err != nil {
-		return nil, err
-	}
-
-	hosts := []*model.APIHost{}
-	for p.hasMore() {
-		resp, err := p.getNextPage(ctx)
-		if err != nil {
-			return nil, err
-		}
-
-		temp := []*model.APIHost{}
-		err = util.ReadJSONInto(resp.Body, &temp)
-		if err != nil {
-			err = resp.Body.Close()
-			if err != nil {
-				return nil, errors.Wrap(err, "error closing response body")
-			}
-			return nil, err
-		}
-
-		hosts = append(hosts, temp...)
-	}
-	return hosts, nil
-}
-
+func (*communicatorImpl) GetAllHosts()     {}
+func (*communicatorImpl) GetHostByID()     {}
```

Comment:
should we remove these if we're not going to implement them?

Reply:
done